source/_integrations/mystrom.markdown

```diff
@@ -17,43 +18,23 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `mystrom` light platform allows you to control your [myStrom](https://mystrom.ch/) WiFi Bulbs.
 
 There is currently support for the following device types within Home Assistant:
 
 - [Light](#light)
+  - [Setup of myStrom Bulbs](#setup-of-mystrom-bulbs)
 - [Binary Sensor](#binary-sensor)
   - [Setup of myStrom Buttons](#setup-of-mystrom-buttons)
 - [Switch](#switch)
   - [Setup](#setup)
 
 ## Light
 
-To use your myStrom WiFi Bulb in your installation, add the following to your `configuration.yaml` file:
+The `mystrom` light platform allows you to control your [myStrom](https://mystrom.ch/) WiFi Bulbs.
```

Comment:
This is now no longer a platform integration. We don't need to backtick `mystrom` anymore in this case.

Reply:
I merged the light and the switch description and removed the backticks